e2e-tests/trade-finance-manager/cypress/e2e/journeys/case/parties/exporter.spec.js

```diff
@@ -61,6 +61,48 @@ context('Parties - user can view and edit exporter', () => {
 
         pages.exporterPage.saveButton().click();
 
+        cy.url().should('eq', relative(`/case/${dealId}/parties/exporter`));
+        pages.exporterPage.errorSummary().contains('Enter a correct party URN');
+        pages.exporterPage.urnError().contains('Enter a correct party URN');
```

Comment:
Can we please add a test case with empty input please.

Reply:
done